documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -3277,53 +3277,74 @@ If you select this option, to prevent the connector from emitting and processing
 
 |[[postgresql-replica-autoset-type]]<<postgresql-replica-autoset-type, `+replica.identity.autoset.values+`>>
 |_empty string_
-|The setting determines the value for link:https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-REPLICA-IDENTITY[replica identity] at table level. +
- +
-This option will overwrite the existing value in database.
-A comma-separated list of regular expressions that match fully-qualified tables and replica identity value to be used in the table. +
- +
-Each expression must match the pattern '<fully-qualified table name>:<replica identity>', where the table name could be defined as (`SCHEMA_NAME.TABLE_NAME`), and the replica identity values are: +
- +
-`DEFAULT` - Records the old values of the columns of the primary key, if any. This is the default for non-system tables. +
- +
-`INDEX index_name` - Records the old values of the columns covered by the named index, that must be unique, not partial, not deferrable, and include only columns marked NOT NULL. If this index is dropped, the behavior is the same as NOTHING. +
- +
-`FULL` - Records the old values of all columns in the row. +
- +
-`NOTHING` - Records no information about the old row. This is the default for system tables. +
- +
-For example,
+|Set this property to apply specific link:https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-REPLICA-IDENTITY[replica identity] settings to a subset of the tables that a connector captures, based on the table name.
+The replica identity values that the property sets overwrite the replica identity values that are set in the database.
+
+This property applies changes to the replica identity of a table only if the connector is configured to capture the table, based on settings in the xref:postgresql-property-table-include-list[`table.include.list`] and xref:postgresql-property-table-exclude-list[`table.exclude.list`] properties.
```

Comment:
Captured tables can be detemined also by using [schema.include.list](https://debezium.io/documentation/reference/3.0/connectors/postgresql.html#postgresql-property-schema-include-list) and/or [schema.exclude.list](https://debezium.io/documentation/reference/3.0/connectors/postgresql.html#postgresql-property-schema-exclude-list)

Reply:
> Captured tables can be detemined also by using [schema.include.list](https://debezium.io/documentation/reference/3.0/connectors/postgresql.html#postgresql-property-schema-include-list) and/or [schema.exclude.list](https://debezium.io/documentation/reference/3.0/connectors/postgresql.html#postgresql-property-schema-exclude-list)

Moved information about the property applying to captured tables to an admonition at the end of the description and modified the text to note the role of the `schema.*.list` properties in determining the tables to capture.